wiki/Modding/Mod_for_mod/id.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+tags:
+  - m4m
+---
+
+# Mod for mod
+
+**Mod for mod** (juga dikenal sebagai *M4M*) adalah sebuah istilah yang menggambarkan kesepakatan antara dua kreator atau lebih. Setiap kreator harus melakukan [mod](/wiki/Modding) pada [beatmap](/wiki/Beatmaps) pengguna lain.
```

Comment:
`kreator` feels really weird here to be honest, given the context here using `mapper` would be a lot better

``**Mod for mod** (juga dikenal sebagai *M4M*) merupakan istilah yang menggambarkan kesepakatan antara dua orang mapper atau lebih, di mana masing-masing mapper sepakat untuk melakukan [modding](/wiki/Modding) terhadap [beatmap](/wiki/Beatmaps) milik mapper lain yang terlibat.``

Reply:
i agree. ok, changed